react/Buttons/Readme.md

```diff
@@ -1,4 +1,10 @@
-The `ref` is forwarded to the root element
```

Comment:
Is it deliberate to remove this line?

Reply:
Yes, I explained why on [the first commit](https://github.com/cozy/cozy-ui/pull/2593/commits/b38b6a02259aa87143677da0f4318db1bc9a6ff0) 😋